Makefile.docker

```diff
@@ -17,8 +17,12 @@ test: $(TEST_IMAGE)
 
 $(TEST_IMAGE):
 	$(eval IMAGE := $(subst _COLON_,:,$@))
-	$(DOCKER) pull $(IMAGE)
-	git archive HEAD | $(DOCKER) run \
-		--cap-add=SYS_ADMIN --rm -i -v /build -w /build -e PY_COLORS=1 \
-		-e TOXENV="$(TOXENV)" -e PYTEST_ADDOPTS="$(PYTEST_ADDOPTS)" $(IMAGE) \
-		/bin/sh -c 'dnf -y install tar && tar -x && dnf -y upgrade openssl-libs && dnf -y install yum && dnf -y install $$(cat test-deps-fedora.txt) && tox'
+	$(DOCKER) build \
+		--build-arg DOCKER_IMAGE="${DOCKER_IMAGE}" \
```

Comment:
My code is using `${DOCKER_IMAGE}`, but `$(IMAGE)` or `$(DOCKER_IMAGE)` is correct?
I do not know the difference of `${FOO}` and `$(FOO)` in `Makefile`.